### DISCUSSION TITLE
[ARCHIVED] Special Topic Meetings

### DISCUSSION THREAD

**Discussion by csarven** [solid/specification#555] [Announcements]
Special Topics Meetings are now being tracked in https://github.com/orgs/solid/projects/16

---

[CG Special Topic Meetings](https://www.w3.org/events/meetings/734bd6fa-e2d6-4254-a725-606244f6e303/
) are tentatively on Tuesdays, 14:00 UTC at https://meet.jit.si/solid-cg with duration 1-2h depending on group's needs and participants. Meetings may be transcribed and [published](https://github.com/solid/specification/tree/main/meetings/).

|Date|Topic|
|-|-|
| 2023-08-08 | [Alternative solutions to container HATEOAS](https://github.com/solid/specification/issues/525) |
| 2023-08-22 | [Solid Demos](https://github.com/solid/specification/blob/main/meetings/2023-08-09.md#Solid-Demos) |
| 2023-08-29 | Cancelled. |
| 2023-09-05 | Server side clients (apps) - Solid OIDC (https://github.com/solid/specification/issues/504#issuecomment-1687212176) 
| 2023-09-12 | Cancelled. |
| 2023-09-19 | Alignment on generalized approach to client-client specs |
| 2023-09-26 | [W3C Solid Community Group and WebAgents Community Group Joint Meeting](https://hackmd.io/ftVbmvj0R0Oh6EsPWTEkwQ) |
| 2023-10-03 | [Alternative solutions to container HATEOAS](https://github.com/solid/specification/issues/525) |
| 2023-10-10 | [Reorganizing CG assets](https://hackmd.io/8Mgs5jt_Ry2656JSOzfkAg). |
| 2023-10-17 | [Solid QA](https://solidproject.org/ED/qa)
| 2023-10-24 | [Security flaws in OIDC spec](https://github.com/solid/solid-oidc/issues/221). |
| 2023-10-31 | [Align shared core terminology](https://github.com/solid/specification/discussions/557) |
| 2023-11-07 | [Security flaws in OIDC spec](https://github.com/solid/solid-oidc/issues/221). |
| 2023-11-14 | [Should access grants edit ACLs?](https://github.com/solid/specification/issues/592) |
| 2023-12-05 | [W3C Solid Community Group and Social Community Group Joint Meeting](https://github.com/solid/specification/blob/main/meetings/2023-12-05.md) |
| 2023-12-12| [Authorization Service/Agent/App](https://github.com/solid/specification/discussions/603) |
| 2024-01-09 |[Chair Election 2023 Retrospective](https://github.com/solid/specification/discussions/582) |
| 2024-01-16 |[Ideas for access modes and corresponding operations in the Protocol](https://github.com/solid/authorization-panel/issues/253) |
| 2024-01-23 |Statement level access control|

This conversation is locked to serve as a reference-board. Please propose a special topic meeting in a [CG weekly call](https://www.w3.org/events/meetings/a97faf55-e599-4fdd-b64b-32db74813e3b/) or in [chat](https://matrix.to/#/#solid_specification:gitter.im).

**Comment by michielbdejong** [comment]
Did this list move to elsewhere? If so we should link to the new location and close this discussion

**Reply by VirginiaBalseiro** [reply]
Yes, it has been a long time ago. I have been announcing it on weekly meeting for the past few months. The board is at https://github.com/orgs/solid/projects/16 . We can see about updating this discussion.

**Reply by VirginiaBalseiro** [reply]
Thanks @csarven for updating!
